test/extensions/filters/listener/original_dst/original_dst_fuzz_test.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.listener.original_dst.v3;
+
+import "envoy/extensions/filters/listener/original_dst/v3/original_dst.proto";
+import "google/protobuf/empty.proto";
+import "validate/validate.proto";
+
+message OriginalDstTestCase {
+  envoy.extensions.filters.listener.original_dst.v3.OriginalDst config = 1;
```

Comment:
It looks like this config field is unused?

Reply:
Yeah, I wasn't sure if I should get rid of it - original_dst doesn't actually have any config options that you can set, but I figured that a fuzzer for a generic listener filter (like original_src) would, so I kept the field for consistency. Should I remove it?

Reply:
Actually, I've decided to just remove all of the unnecessary dependencies (including this config field).

Reply:
Yeah I think so. It marginally increases the fuzzed input space. 

Generic listener filters will probably consume from a generic proto anyway instead of `OriginalDstTestCase` or `OriginalDst config`, so that's a proto to design and define later :)